tests/Integration/RestAPI/ContentHelper/ContentHelperFeatureTestTrait.php

```diff
@@ -0,0 +1,293 @@
+<?php
+/**
+ * Trait for testing the Content Helper API feature.
+ *
+ * @package Parsely
+ * @since   3.17.0
+ */
+
+namespace Parsely\Tests\Integration\RestAPI\ContentHelper;
+
+use Parsely\Permissions;
+use Parsely\REST_API\Base_Endpoint;
+use Parsely\Tests\Integration\TestCase;
+use WP_Error;
+use WP_REST_Request;
+
+/**
+ * Trait for testing the Content Helper API feature.
+ *
+ * @since 3.17.0
+ *
+ * @covers \Parsely\REST_API\Content_Helper\Content_Helper_Feature
+ */
+trait ContentHelperFeatureTestTrait {
+
+
+	/**
+	 * Test that the endpoint is available to the current user.
+	 *
+	 * @since 3.17.0
+	 *
+	 * @covers \Parsely\REST_API\Content_Helper\Content_Helper_Feature::is_available_to_current_user
+	 * @uses Parsely\Endpoints\Base_Endpoint::__construct
```

Comment:
Missing leading `\`, although they might be unnecessary. For the time being, let's add them by convention, and we can see if we can/should remove them in the future. Worth checking to see if any of these are missing in any other tests.

```suggestion
	 * @uses \Parsely\Endpoints\Base_Endpoint::__construct
```